src/Internal/Plutip/Types.purs

```diff
@@ -119,8 +118,7 @@ newtype ClusterStartupRequest =
     , epochSize :: Maybe UInt
     , slotLength :: Maybe Seconds
     , maxTxSize :: Maybe UInt
-    , increasedExUnits :: Maybe UInt
-    , noCollateral :: Boolean
+    , raiseExUnitsToMax :: Maybe Boolean
```

Comment:
If this is a `Maybe` here, can we make it a `Maybe` in `ClusterConfig` too to provide a way for users to say "the default plutip value"?